include/onnxruntime/core/graph/graph.h

```diff
@@ -1139,16 +1139,40 @@
   const ONNX_NAMESPACE::GraphProto& ToGraphProto();
   ONNX_NAMESPACE::GraphProto ToGraphProto() const;
 
+  // Options to align external initializer offset.
+  struct OffsetAlignmentInfo {
+    // Offset will always be page aligned and allocation granularity aligned for mmap support.
+    // This is done by padding previous tensor data with zeros keeping same length.
```

Comment:
From the comments it is not clear what effect align_offset has if any.

Reply:
Will refine comments, thanks